assets/css/_detour_list_page.scss

```diff
@@ -0,0 +1,8 @@
+.c-detour-list-page {
+  background-color: white;
+}
+
+.c-detour-list-page__button {
+  padding: 0.5rem 1rem 0.5rem 1rem;
+  margin: 24px 24px 36px 24px;
+}
```

Comment:
I love css, but is there a reason to do this and add to the bundle size vs using bootstrap utilities?

Reply:
Couple thoughts:

1. I forgot to push a fix I had where I swapped all of these `px`'s for `rem`'s, so this should actually be `margin-bottom: 2.25rem`.
2. I can replace almost all of these with bootstrap utilities, except for `margin-bottom: 2.25rem`. There's no bootstrap utility for `2.25 * $spacer`.
3. BUT... I think it still looks pretty good with `mb-4`, which is `1.5 * $spacer`, so maybe we can ditch the CSS entirely for now.

<img width="691" alt="Screenshot 2024-09-03 at 12 42 51 PM" src="https://github.com/user-attachments/assets/278ca0c1-c5f5-458d-84ae-4c99cedc1abd">

Reply:
I changed my mind and added back in a teeny bit of CSS that enables the bigger margin. I replaced everything else with utilities though.